app/controllers/entries_controller.rb

```diff
@@ -13,8 +13,19 @@ def show
   end
 
   def new
+    @entry = Entry.new
   end
 
   def edit
   end
+
+  def create
+    # fail
+    @entry = Entry.new(
```

Comment:
This variable should just be entry, not @entry. @entry makes it an instance variable which gets passed to the view and can be rendered. In this case, this variable is never used outside this method

Reply:
Isn't it being used in the new.html.erb?
I realize new.html.erb could be written without it, but if we plan on having an Edit Phrase page, shouldn't it need the instance variable?

Reply:
I forgot to mention, I would end up making the new.html.erb use a partial form which is shared with edit.html.erb